Source/Core/UICommon/AutoUpdate.cpp

```diff
@@ -80,6 +80,18 @@ void CleanupFromPreviousUpdate()
 #else
   File::Delete(reloc_updater_path);
 #endif
+
+#ifdef __APPLE__
+  std::string embedded_updater_path =
+    File::GetBundleDirectory() + "/Contents/Helpers/" + UPDATER_FILENAME;
+  if (File::Exists(embedded_updater_path))
+  {
+    // Starting with PR 10428, the updater app is now embeded in the main bundle on universal builds.
```

Comment:
I don't think we need "on universal builds". All macOS builds are universal for the time being, and even when we eventually start making ARM only builds this change would apply for them too.